download_models/downloadOVMSModels.sh

```diff
@@ -265,4 +256,15 @@ for eachModelBasePath in "${model_base_path[@]}" ; do
     else
         echo "$eachModel model already exists, skip downloading..."
     fi
-done
\ No newline at end of file
+done
+
+# FP32-INT8 efficientnet-b0 for capi
+customefficientnetb0Modelfile="$efficientnetb0/FP32-INT8/1/efficientnet-b0.xml"
+if [ ! -f $customefficientnetb0Modelfile ]; then
+    mkdir -p "$efficientnetb0"
+    mkdir -p "$efficientnetb0/FP32-INT8"
+    mkdir -p "$efficientnetb0/FP32-INT8/1"
+
+    wget "https://github.com/dlstreamer/pipeline-zoo-models/raw/main/storage/efficientnet-b0_INT8/FP32-INT8/efficientnet-b0.bin" -P "$efficientnetb0/FP32-INT8/1"
+    wget "https://github.com/dlstreamer/pipeline-zoo-models/raw/main/storage/efficientnet-b0_INT8/FP32-INT8/efficientnet-b0.xml" -P "$efficientnetb0/FP32-INT8/1"
```

Comment:
why this new addition?  Couldn't we use the existing omz downloader??

Reply:
just use the existing omz_downloader to download if not the same, we could not do that?

Reply:
I tried it before, existing omz_downloader doesn't support this version of efficientnet-b0

Reply:
not sure - maybe we can pair to see